src/exception/missing-entity-exceptions/api.ts

```diff
@@ -22,9 +22,20 @@ import { Request, Response } from 'express';
 import { parseBoolString } from '../../utils/request';
 import * as service from './service';
 import { z as zod } from 'zod';
-import { listAll } from './repo';
+import { getByProgramId, listAll } from './repo';
 
 class MissingEntityExceptionController {
+	@HasFullReadAccess()
+	async getProgramException(req: Request, res: Response) {
+		const result = await getByProgramId(req.params.programId);
+		if (!result.success) {
+			const { message, errors } = result;
+			return res.status(500).send({ message, errors });
+		} else {
```

Comment:
When we have no database entry, can we instead reply with a blank exception object?
example:
```ts
{
  programId: "SomeId",
  donorSubmitterIds: []
}
```

Reply:
Also note for reference... if the response shows no entry for the parameters the user requested, thats a 4xx error of some sort, based on user inputs, not a 5xx server error.

Reply:
If there's no db entry that's happening already https://github.com/icgc-argo/argo-clinical/blob/feat%2Fmissing-entity-exception/src/exception/missing-entity-exceptions/repo.ts#L84

Reply:
in this case though it is sent with status 200 and the object you suggested above

Reply:
Yes that is correct lol. Already handled.